js/repl/CodeMirrorPanel.js

```diff
@@ -4,6 +4,7 @@ import { css } from "emotion";
 import CodeMirror from "./CodeMirror";
 import React from "react";
 import { colors } from "./styles";
+import prettyFormat from "pretty-format";
```

Comment:
Should we be using this or https://github.com/pretty-format/pretty-format2 @thejameskyle

Reply:
My preference would be neither, unless we're loading from a CDN. This package adds 16.1kB gzipped to the bundle.

(`pretty-format2` would only add 8kB gzipped, but still)

Reply:
we can use a cdn for sure, otherwise we run it through json.stringify